doc/governance.rst

```diff
@@ -58,6 +58,7 @@ Current CLT members are:
  * Gregory Farnum <gfarnum@redhat.com>
  * Haomai Wang <haomai@xsky.com>
  * Jason Dillaman <dillaman@redhat.com>
+ * Patrick Donnelly <pdonnell@redhat.com>
```

Comment:
nit, the names seem to be in alphabetical order

Reply:
Thanks, fixed!